MAUI/DataGrid/pull-to-refresh.md

```diff
@@ -10,6 +10,135 @@ keywords : maui datagrid, maui grid, grid maui, maui gridview, grid in maui, .ne
 
 # Pull To Refresh in MAUI DataGrid (SfDataGrid)
 
+The data grid enables the `PullToRefresh` option by setting the [SfDataGrid.AllowPullToRefresh]() property to `true`, the control supports for refreshing the data source at runtime while doing the pull to refresh action. 
+
+## Pull to refresh command
+
+The data grid refreshes the data in view at runtime by triggering an `ICommand` bound to the [SfDataGrid.PullToRefreshCommand]() property. When you perform a pull-to-refresh action, if the progress bar reaches 100%, this command is triggered to refresh the records in view.
+
+Set the [SfDataGrid.IsBusy]() property to `true` before refreshing the records to notify the grid that a pull-to-refresh action is being performed, and set the property to `false` after the view is refreshed. You can also adjust the pull-to-refresh animation time by setting a delay in the sample.
+
+To enable and perform the pull-to-refresh operation, follow the code example:
```

Comment:
The data grid can refresh its data during runtime by executing an `ICommand` bound to the [SfDataGrid.PullToRefreshCommand]() property. When a pull-to-refresh gesture is performed and the progress bar reaches 100%, this command is triggered to update the records in view.

To indicate that a pull-to-refresh operation is in progress, set the [SfDataGrid.IsBusy]() property to true before starting the refresh. Once the refresh is complete, set this property back to false. You can also customize the duration of the pull-to-refresh animation by adding a delay in the code.

To enable and perform the pull-to-refresh operation, refer to the following code example:

Reply:
modified